### ISSUE TITLE
save reports from pad

### ISSUE THREAD

**Issue by jaromil** [dyne/reflow#1]
Beware that pads (the reports linked at end of README) aren't guaranteed to last forever: content should be saved in this repository if important.

**Comment by aspasia** [comment]
Good point, you ok with this https://github.com/dyne/reflow/commit/2c942497075e03f12450ded0ec6bd28455d21a66
?

**Comment by jaromil** [comment]
Perhaps better remove the link to a private platform (to avoid spam, bots etc. indexing) and just mention we have on cloud in folder 2019_REFLOW document names [list, date, place].
